src/12_scopes.py

```diff
@@ -4,8 +4,11 @@
 # When you use a variable in a function, it's local in scope to the function.
 x = 12
 
+
 def changeX():
     x = 99
+    print(x)
```

Comment:
I guess with this code we are creating new x variable inside the scope of changeX() function

But we need to change the value of the x outside of the scope of changeX() function.